azure-pipelines.yml

```diff
@@ -37,29 +37,6 @@ stages:
   dependsOn: Check
   jobs:
 
-  - job: Lint
```

Comment:
This was indeed fairly pointless duplication of a lint job, and the `Python.h` check fits better on Linux, so this can be removed.

Reply:
This is being done in #30225